t/lib/Test/Resty.pm

```diff
@@ -87,6 +87,18 @@ sub run_test ($) {
         $cmd .= " $luafile"
     }
 
+    my $perlfile;
+    if (defined $block->perlsrc) {
+        my ($out, $perlfile) = tempfile("testXXXXXX",
+                                        SUFFIX => '.pl',
```

Comment:
Let's make it more general. It does not have to be Perl script so it does not have to use a `.pl` file name suffix.